packages-exp/firebase-exp/messaging/index.ts

```diff
@@ -14,4 +14,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export * from '@firebase/messaging-exp';
+export * from '@firebase/messaging-exp/dist/index';
```

Comment:
I think it might be better to split the regular and sw versions using a subdirectory instead of file name patterns. I think you can put a subdirectory like `packages-exp/messaging-exp/sw` and put a `package.json` in it whose `main`, `browser`, `module`, `typings` point to the right files in `dist`.

Then this line can go back to regular `from '@firebase/messaging-exp'` and the cdn export can be `from '@firebase/messaging-exp/sw'`. I think that's the more idiomatic way.